test/src/org/apache/jorphan/util/TestJorphanUtils.java

```diff
@@ -262,6 +268,13 @@ public void testbaToHexString(){
         assertEquals("0f107f8081ff",JOrphanUtils.baToHexString(new byte[]{15,16,127,-128,-127,-1}));
     }
 
+    @Test
+    public void testbaToHexStringSeparator(){
```

Comment:
This could be test**B**aToHexStringSeparator()
The previous existing test also has a lowercase `b` at the start of the method name but almost all others have a capital latter here. Just saying, not critical imo.

Reply:
Done